### PR TITLE
Technique example: data loader, R to JPEG

### PR DESCRIPTION
**Pull request by allisonhorst**
Deployed: https://observablehq.observablehq.cloud/framework-example-loader-r-to-jpeg/

### CODE REVIEW

**Review by mbostock — COMMENTED** [review]
Funny enough but I don’t mind the “a JPEG” here and I can’t say why. 😅 

Still, can you generalize the feedback I gave on the R-to-JSON pull request to apply here also? Here I notice a file without a trailing newline, and this example should also be listed in `examples/README.md`. Thanks.

File: examples/loader-r-to-jpeg/src/index.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+To run this data loader, you’ll need R and the `ggplot2` package installed, _e.g._ using `install.packages("ggplot2")`.
+
+</div>
+
+The above data loader lives in `data/diamonds.jpeg.R`, so we can load the data using `data/diamonds.jpeg`. Access the image in a markdown page using FileAttachment:
+
+```js echo
+const diamonds = FileAttachment("data/diamonds.jpeg").image({width: 500});
+```
```
Comment (mbostock) [inline examples/loader-r-to-jpeg/src/index.md:33]: Would be it easier to just use a static image here?

```md
<img src="data/diamonds.jpeg" style="max-width: 500px;">
```

**Comment by allisonhorst** [comment]
> can you generalize the feedback I gave on the R-to-JSON pull request to apply here also? Here I notice a file without a trailing newline, and this example should also be listed in `examples/README.md`. Thanks.

Yeah will do, I'll ping you once I've made the updates to all of these open technique example PRs!

File: examples/loader-r-to-jpeg/src/index.md (hunk 1 above)
Comment (allisonhorst) [reply examples/loader-r-to-jpeg/src/index.md:33]: Yep that makes sense, I'll switch it out

**Review by Fil — APPROVED** [review]

**Pull request merged**